netbout-client/pom.xml

```diff
@@ -189,6 +189,28 @@
                                  Findbugs filters in current version of Qulice
                                 -->
                                 <exclude>findbugs:~com.netbout.client.*</exclude>
+                                <!--
+                                 Additional to puzzle #961:30min declared in
```

Comment:
@danff this is to be removed, and appropriate comment added to the other puzzle (just mention that in netbout-client/pom.xml and checkstyle and pmd excludes should bre removed), you could also create two puzzles, if you think that the work in netbout-client is for 30mins and in netbout-web is also for 30mins.
You just have to make the text different.